projects/swimlane/ngx-charts/src/lib/pie-chart/pie-chart.component.ts

```diff
@@ -129,7 +130,7 @@ export class PieChartComponent extends BaseChartComponent {
 
     // sort data according to domain
     this.data = this.results.sort((a, b) => {
-      return this.domain.indexOf(a.name) - this.domain.indexOf(b.name);
+      return this.domain.indexOf(a.label) - this.domain.indexOf(b.label);
```

Comment:
With the specified typings, there is no `a.name` available. Was this an error or did I miss maybe something 🤔?